cmd/kubectl-gadget/deploy.go

```diff
@@ -207,6 +215,8 @@ func parseK8sYaml(content string) ([]runtime.Object, error) {
 
 	// For CustomResourceDefinition kind.
 	apiextv1.AddToScheme(sch)
+	// For ClusterImagePolicy kind.
+	sigstoreapi.AddToScheme(sch)
```

Comment:
Instead of importing this with all of its dependencies for a simple type test, we could also just add a dummy like so
```go
sch.AddKnownTypeWithName(schema.GroupVersionKind{Group: "policy.sigstore.dev", Version: "v1beta1", Kind: "ClusterImagePolicy"}, &unstructured.Unstructured{})
```

And then later on instead of doing the casting check, you could just compare `object.GetObjectKind().GroupVersionKind()` against the corresponding values.

If we need the imports for other things as well later on, then of course this makes no difference.